src/Draco.Fuzzer/Helper.cs

```diff
@@ -0,0 +1,30 @@
+using System.Diagnostics;
+using System.Text;
+
+namespace Draco.Fuzzer;
+
+internal static class Helper
```

Comment:
Since you have a base-class for fuzzers, why not move the responsibility of catching the errors there?